src/components/BookActions.js

```diff
@@ -8,7 +8,19 @@ const BookActions = (props) => {
   const { bookid } = props;
 
   const removeBookFromStore = () => {
-    dispatch(removeBook(bookid));
+    const deleteBooks = () => (dispatch) => {
+      dispatch(removeBook(bookid));
+      fetch(`https://us-central1-bookstore-api-e63c8.cloudfunctions.net/bookstoreApi/apps/oWMP1iS5uMQBHYqtCwUz/books/${bookid}`, {
+        method: 'DELETE',
+        body: JSON.stringify({ item_id: `${bookid}` }),
+        headers: {
+          'Content-type': 'application/json; charset=UTF-8',
+        },
+      })
+        .then((response) => response);
+    };
+
+    dispatch(deleteBooks());
```

Comment:
- [ ] It would be better for the books that delete update the API. Currently, I cannot delete the books from the API it just deletes them from the app, once the window is refreshed the books appear again. Kindly fix this. 
😄 💯